src/js/ublock.js

```diff
@@ -462,11 +559,16 @@ var matchWhitelistDirective = function(url, hostname, directive) {
     µBlock.goodblock.browserState.adTabId = null;
 
     µBlock.goodblock.goodnightGoodblock();
+    µBlock.goodblock.log.logEvent('adClosed');
 }
 
 /******************************************************************************/
 
 µBlock.goodblock.markIfGoodblockIsAwake = function(isAwake) {
+    if (!µBlock.goodblock.browserState.isAwake && isAwake) {
+      // When tranisitioning from asleep to awake, set last wake time
+      µBlock.goodblock.browserState.lastWakeTime = new Date().getTime();
+    }
```

Comment:
Delete these additions. This logic moved elsewhere, so we don't need this block.